packages/apps/shopify-app-remix/src/server/config-types.ts

```diff
@@ -163,6 +163,9 @@ export interface AppConfigArg<
   /**
    * How your app is distributed. Default is `AppDistribution.AppStore`.
    *
+   * SingleMerchant should be used for custom apps managed in the Partner Dashboard.
+   * ShopifyAdmin should be used for apps that are managed in the merchant's Shopify Admin.
```

Comment:
do we want to include some explanation of the default value `AppDistribution.AppStore`?